### PR TITLE
RU localization files

### PR DESCRIPTION
**Pull request by Sidiusz**
1. Added the link to the official site, removed the community VK link
2. Fixed Russian and other symbols in the font
3. Added Russian localization file

### CODE REVIEW

File: Code/skyrim_ui/src/app/transloco-root.module.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export class TranslocoHttpLoader implements TranslocoLoader {
         availableLangs: [
           { id: 'en', label: 'English' },
           { id: 'de', label: 'Deutsch' },
+		  { id: 'ru', label: 'Русский' },
```
Comment (RobbeBryssinck) [inline Code/skyrim_ui/src/app/transloco-root.module.ts:29]: Indentation

**Comment by RobbeBryssinck** [comment]
Why the change in font file?

**Comment by Sidiusz** [comment]
> Why the change in font file?

Cyrillic letters do not have the same style as Latin letters

File: Code/skyrim_ui/src/assets/i18n/ru.json [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    "DISCONNECT": {
+      "PROMPT": "Вы уверены, что хотите отключиться?",
+      "PROCEED": "Подтвердить",
+      "CANCEL": "Отмена"
+    },
+    "ERROR": {
+      "OK": "Ок"
+    },
+    "NOTIFICATIONS": {
+      "MESSAGE_FROM": "Сообщение от "
```
Comment (scomans) [inline Code/skyrim_ui/src/assets/i18n/ru.json:35]: missing:
```
"ACCEPT": "Accept",
"DECLINE": "Decline",
```

**Review by scomans — CHANGES_REQUESTED** [review]

File: Code/skyrim_ui/src/assets/i18n/ru.json [hunk 3]
```diff
@@ -1,155 +1,157 @@
...
+      "PROCEED": "Подтвердить",
+      "CANCEL": "Отмена"
+    },
+    "ERROR": {
+      "OK": "Ок"
+    },
+    "NOTIFICATIONS": {
+      "MESSAGE_FROM": "Сообщение от ",
+      "ACCEPT": "Принять",
+      "DECLINE": "Отклонить"
```
Comment (Sidiusz) [inline Code/skyrim_ui/src/assets/i18n/ru.json:37]: Added

**Pull request merged**